.github/ISSUE_TEMPLATE/issue_report.yml

```diff
@@ -0,0 +1,216 @@
+name: Issue Report
+description: File a report for ROCm related issues on Linux and Windows. For issues pertaining to documentation or non-bug related, please open a blank issue located below.
+title: "[Issue]: "
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for taking the time to fill out this report!
+
+      You can acquire your OS, CPU, GPU (for filling out this report) with the following commands:
```

Comment:
Recommended

You must provide details about the OS, CPU, and GPU in the report and may use the following commands to gather additional information for Linux and Windows.